ftw/news/browser/news_listing.py

```diff
@@ -0,0 +1,175 @@
+from Acquisition import aq_parent, aq_inner
+from DateTime import DateTime
+from DateTime.interfaces import SyntaxError as dtSytaxError
+from ftw.news import _
+from ftw.news import utils
+from ftw.news.interfaces import INewsListingView
+from plone.portlets.interfaces import IPortletAssignmentMapping
+from plone.portlets.interfaces import IPortletManager
+from plone.portlets.interfaces import IPortletRenderer
+from Products.CMFCore.permissions import AccessInactivePortalContent
+from Products.CMFCore.utils import _checkPermission
+from Products.CMFCore.utils import getToolByName
+from Products.CMFPlone.interfaces import IPloneSiteRoot
+from Products.CMFPlone.PloneBatch import Batch
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope.component import getMultiAdapter
+from zope.component import getUtility
+from zope.component import queryMultiAdapter
+from zope.interface import implements
+from zope.publisher.browser import BrowserView
+
+
+class NewsListing(BrowserView):
+    implements(INewsListingView)
+    template = ViewPageTemplateFile('templates/news_listing.pt')
+
+    def __init__(self, context, request):
+        super(NewsListing, self).__init__(context, request)
+        self.batch = None
+
+    def __call__(self):
+        b_start = self.request.form.get('b_start', 0)
+        self.batch = Batch(self.get_items(), 10, b_start)
+        return self.template()
+
+    def get_items(self):
+        query = {
+            'object_provides': 'ftw.news.interfaces.INews',
+            'sort_on': 'effective',
+            'sort_order': 'reverse'
+        }
+        show_inactive = _checkPermission(AccessInactivePortalContent,
+                                         self.context)
```

Comment:
Since the whole package is based on the news of ftw.contentpage, I have taken this bit from https://github.com/4teamwork/ftw.contentpage/blob/ea46dd63f0f65451ed9e7372442319e2b90699ae/ftw/contentpage/browser/baselisting.py#L63
Should I remove ist?